### PR TITLE
[rocket_launch_0]: First PR for test

### PR DESCRIPTION
**Pull request by hu-minghao**
- brief: First PR for test

- author: 530019431@qq.com

### What changes are included in this PR?
 新建小组文件夹，增加个人文件夹并放入测试代码。

### Are these changes tested?
I think so

### CODE REVIEW

**Comment by Light-City** [comment]
> * brief: First PR for test
> * author: [530019431@qq.com](mailto:530019431@qq.com)
> 
> ### What changes are included in this PR?
> 新建小组文件夹，增加个人文件夹并放入测试代码。
> 
> ### Are these changes tested?
> I think so

format it use markdown docs.

**Comment by Light-City** [comment]
directory rule is wrong. please check.

**Comment by hu-minghao** [comment]
modify dir style and add markdown docs.
[FirstPR.md](https://github.com/datawhalechina/whale-starry/files/11778065/FirstPR.md)

**Comment by Light-City** [comment]
FirstPR.md -> firstpr or first_pr.md

lowercase underscore style.

**Comment by hu-minghao** [comment]
ok,modify

File: stl/rocket_launch/beichen/src/FirstPR.md
Comment (martinx) [inline stl/rocket_launch/beichen/src/FirstPR.md:1]: Please remove, as you already committed the renamed file.

File: stl/rocket_launch/beichen/src/hello_world.cpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <vector>
+#include <string>
+
+int main() {
+    std::vector<char> data;
```
Comment (martinx) [inline stl/rocket_launch/beichen/src/hello_world.cpp:6]: Indent 2 spaces.

**Comment by martinx** [comment]
pr title should be in the following format:

team_[team_name]_

```
[rocket_launch_0] =>  [team_rocket_launch_0]
```

**Pull request closed without merging**